README.md

```diff
@@ -15,7 +15,7 @@ This information reflects the head of this branch.
 
 | Compatible with CSI Version                                                                | Container Image                                  | [Min K8s Version](https://kubernetes-csi.github.io/docs/kubernetes-compatibility.html#minimum-version) |
 | ------------------------------------------------------------------------------------------ | ------------------------------------------------ | ------------------------------------------------------------------------------------------------------ |
-| [CSI Spec v1.3.0](https://github.com/container-storage-interface/spec/releases/tag/v1.3.0) | k8s.gcr.io/sig-storage/csi-node-driver-registrar | 1.13                                                                                                   |
+| [CSI Spec v1.5.0](https://github.com/container-storage-interface/spec/releases/tag/v1.5.0) | k8s.gcr.io/sig-storage/csi-node-driver-registrar | 1.20                                                                                                   |
```

Comment:
k8s version should stay at 1.13

Reply:
hey @humblec, could you please take a look into this? After this is merged I'll work on the release

Reply:
@msau42 @mauriciopoppe done..